cmd/contour/servecontext.go

```diff
@@ -49,13 +49,12 @@ type serveContext struct {
 	Kubeconfig string `yaml:"kubeconfig,omitempty"`
 
 	// contour's xds service parameters
-	xdsAddr string
-	xdsPort int
-	// Defines the XDSServer to use for `contour serve`
-	// Defaults to "contour"
-	XDSServerType                   string `yaml:"xds-server-type,omitempty"`
+	xdsAddr                         string
+	xdsPort                         int
```

Comment:
does it make sense for these (and/or any others) to move as well? need to keep in mind compat issues if moving any exported fields

Reply:
Yup I think they should, just a matter of doing that and starting up the deprecation cycle. I can make a new issue to track this work maybe rather than doing in this PR?

Reply:
sure thing. Any nonexported fields should be able to just move (I think), since they're not settable via config file.

Reply:
Nope you're right, I was overthinking all this deprecation stuff. I'll move them in now.